docs/controls/WebView-known-issues.md

```diff
@@ -0,0 +1,112 @@
+
+* [LongRunningScriptDetected](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.longrunningscriptdetected)
+
+* [UnsafeContentWarningDisplaying](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.unsafecontentwarningdisplaying)
+
+* [UnsupportedUriSchemeIdentified](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.unsupportedurischemeidentified)
+
+* [UnviewableContentIdentified](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.unviewablecontentidentified)
+
+* [NewWindowRequested](https://docs.microsoft.com/uwp/api/windows.web.ui.iwebviewcontrol.newwindowrequested)
+
+## Rendering and layout
+
+* The **WebView** control doesn't appear at the correct scale when users move the host application between monitors that have different screen resolutions.
+
+* The **WebView** control doesn't appear as expected on some high-resolution displays.
+
+* The [Control.Bounds](https://msdn.microsoft.com/library/system.windows.forms.control.bounds.aspx) property is not supported.
+
+## Performance
+
+* **WebView** controls in WPF applications have longer load times than Windows Forms applications.
+
+* If the web view hangs or stops working, other applications that use the same underlying process (Win32WebViewHost) will hang or stop working as well.
+
+## Security
+
+* **WebView** controls don't render in XAML Browser Applications (XBAP).
+
+  That's because a browser has partial trust to the system while WebView controls have full-trust.
+
+* **WebView** controls won't appear if the Win32WebViewHost application is disabled in the location settings of the user's system.
+
+* **WebView** controls won't function as expected in a WPF-based ClickOnce application.
+
+* **WebView** controls function only in applications that use a Single-Threaded Apartment (STA) model. Multithreaded Apartment (MTA) model is not supported.
+
+* **WebView** controls have not been tested for Windows Information Protection. Therefore, the information that moves in and out of the WebView control might not be properly secured even if an  organizational administrator applies a Windows Information Protection policy to the hosting application.  
+
+* **WebView** controls have not been exhaustively tested for security.
+
+## User interaction
+
+* Accelerator keys don't work in **WebView** controls.
+
+* Users can't use the TAB key to put the **WebView** into focus. They'll have to use the mouse or other pointing device to put the **WebView** control into focus.
+
+## Paths and strings
+
+* You can use only fully qualified paths to resources in members of the **WebView** control that accept string paths. **WebView** controls don't recognize the File:// or ms-appx:/// prefixes so they can't read from the file system or the package (if you've created a package for your application).
```

Comment:
We only accept absolute URIs. Workaround for reading files is to read the content in, serialize to a string, and use `void NavigateToString(String)` method